tests/tests.toml

```diff
@@ -229,3 +229,8 @@ tokens = true
 [[test]]
 filename = "tokens/indent1.py"
 tokens = true
+
+[[test]]
+filename ="../integration_tests/test_max.py"
```

Comment:
I think this file does not exist in the PR. Did you forget to include it?

Reply:
i included but i made conflict that remove things.